src/frontend/handlers.go

```diff
@@ -40,7 +40,8 @@ type platformDetails struct {
 }
 
 var (
-	templates = template.Must(template.New("").
+	isCymbalBrand = os.Getenv("CYMBAL_LOGO") == "true"
```

Comment:
**nit:** Should we account for case-sensitivity? `True` and `TRUE` or other combinations?
```suggestion
	isCymbalBrand = os.Getenv("CYMBAL_LOGO", "false").lower() == "true"
```

Reply:
Done!
```
isCymbalBrand = "true" == strings.ToLower(os.Getenv("CYMBAL_BRANDING"))
```